src/include/ipc4/copier.h

```diff
@@ -70,16 +70,6 @@ struct ipc4_base_module_cfg_ext {
 	struct ipc4_output_pin_format output_pins[1];
 } __attribute__((packed, aligned(4)));
 
-/* This enum defines short 16bit parameters common for all modules.
- * Value of module specific parameters have to be less than 0x3000.
- */
-enum ipc4_base_module_params {
-	/* handled inside LargeConfigGet of module instance */
-	IPC4_MOD_INST_PROPS  = 0xFE,
```

Comment:
Please note we have below: `struct ipc4_module_instance_props`
It is binded to IPC4_MOD_INST_PROPS definition. It should be moved together.